[2024-12-26T04:28:27.857+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T04:28:27.865+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9868205040_2024-12-26T04-27-58-728021Z.update_schedule_flag scheduled__2024-12-26T04:27:58.728021+00:00 [queued]>
[2024-12-26T04:28:27.868+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9868205040_2024-12-26T04-27-58-728021Z.update_schedule_flag scheduled__2024-12-26T04:27:58.728021+00:00 [queued]>
[2024-12-26T04:28:27.868+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-26T04:28:27.874+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): update_schedule_flag> on 2024-12-26 04:27:58.728021+00:00
[2024-12-26T04:28:27.879+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=118) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-26T04:28:27.879+0000] {standard_task_runner.py:72} INFO - Started process 119 to run task
[2024-12-26T04:28:27.880+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9868205040_2024-12-26T04-27-58-728021Z', 'update_schedule_flag', 'scheduled__2024-12-26T04:27:58.728021+00:00', '--job-id', '949', '--raw', '--subdir', 'DAGS_FOLDER/schedule_call.py', '--cfg-path', '/tmp/tmpbltmf4rs']
[2024-12-26T04:28:27.880+0000] {standard_task_runner.py:105} INFO - Job 949: Subtask update_schedule_flag
[2024-12-26T04:28:27.904+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9868205040_2024-12-26T04-27-58-728021Z.update_schedule_flag scheduled__2024-12-26T04:27:58.728021+00:00 [running]> on host 680cb867086b
[2024-12-26T04:28:27.943+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9868205040_2024-12-26T04-27-58-728021Z' AIRFLOW_CTX_TASK_ID='update_schedule_flag' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T04:27:58.728021+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-26T04:27:58.728021+00:00'
[2024-12-26T04:28:27.943+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T04:28:27.948+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2024-12-26T04:28:27.954+0000] {sql.py:544} INFO - Running statement: UPDATE user_details SET status = 'answered', modified_on_utc = now() at time zone 'utc' WHERE id = 209, parameters: None
[2024-12-26T04:28:27.955+0000] {sql.py:553} INFO - Rows affected: 1
[2024-12-26T04:28:27.957+0000] {logging_mixin.py:190} INFO - Marked record 209 as scheduled.
[2024-12-26T04:28:27.957+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-26T04:28:27.962+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T04:28:27.962+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9868205040_2024-12-26T04-27-58-728021Z, task_id=update_schedule_flag, run_id=scheduled__2024-12-26T04:27:58.728021+00:00, execution_date=20241226T042758, start_date=20241226T042827, end_date=20241226T042827
[2024-12-26T04:28:27.973+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T04:28:27.981+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T04:28:27.982+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
